private-templates-service/client/src/components/NavBar/styled.tsx

```diff
@@ -59,7 +59,7 @@ export const Styledh1 = styled.div`
   }
 `;
 
-export const Styledh2 = styled.div`
+export const Styledh2 = styled.h2`
```

Comment:
Can you please confirm that changing this to h2 didn't make something that was not bold become bold?

Reply:
Yep, nothing changed to bold!